pymc/model.py

```diff
@@ -465,52 +465,51 @@ def __init__(self, mean=0, sigma=1, name=''):
                 # variables in several ways note, that all variables
                 # will get model's name prefix
 
-                # 3) you can create variables with Var method
-                self.Var('v1', Normal.dist(mu=mean, sigma=sd))
-                # this will create variable named like '{prefix::}v1'
+                # 3) you can create variables with the register_rv method
+                self.register_rv(Normal.dist(mu=mean, sigma=sigma), 'v1', initval=1)
```

Comment:
was the initval needed?

Reply:
There are other more important arguments that can be passed to `register_rv`. I don't know what was the rationale to show this one in particular.

Reply:
But we can keep it as another example below shows the initval as well

Reply:
Allright sounds good